reana-ui/src/components/WorkflowBadges.js

```diff
@@ -0,0 +1,66 @@
+import styles from "./WorkflowBadges.module.scss";
+import { Label } from "semantic-ui-react";
+import { JupyterNotebookIcon } from "~/components";
+import { INTERACTIVE_SESSION_URL } from "~/client";
+import { LauncherLabel } from "~/components";
+import { getReanaToken } from "~/selectors";
+import { useSelector } from "react-redux";
+import { statusMapping } from "~/util";
+
+export default function WorkflowBadges({ workflow }) {
+  const reanaToken = useSelector(getReanaToken);
+  const {
+    id,
+    size,
+    launcherURL,
+    session_uri: sessionUri,
+    session_status: sessionStatus,
+  } = workflow;
+  const hasDiskUsage = size.raw > 0;
+  const isSessionOpen = sessionStatus === "created";
+
+  return (
+    <>
+      <div className={styles.badgesContainer}>
+        {workflow.duration && (
+          <Label
+            size="tiny"
+            content={workflow.duration}
+            icon="clock"
+            as="a"
+            href={"/details/" + id}
+            target="_blank"
+            rel="noopener noreferrer"
+            color={statusMapping[workflow.status].color}
+            onClick={(e) => e.stopPropagation()}
+          />
+        )}
+        {hasDiskUsage && (
+          <Label
+            size="tiny"
+            content={size.human_readable}
+            icon="hdd"
```

Comment:
Another functional observation. Clicking on the workspace size brings people to the default detailed record page. It would be nicer if the user landed on the workspace tab on the detailed page, because people clicking on the workspace size would probably like to consult workspace content directly.

Reply:
See previous comment